packages/doc-ai-react/src/App.css

```
@@ -1,44 +1,1190 @@
-#root {
+/* Global Styles */
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: 'Inter', -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Helvetica, Arial, sans-serif;
```

Stylelint (17.6.0)

[error] Unexpected quotes around "Inter" (font-family-name-quotes)

(font-family-name-quotes)